pkg/fetcher/fetcher.go

```
@@ -235,3 +236,14 @@ func (f *Fetcher) FetchBlocks(ctx context.Context, startBlockInclusive uint64, e
 
 	return fetchedBlocks, nil
 }
+
+func (f *Fetcher) GetContractStorageSlot(ctx context.Context, contractAddress string, blockNumber uint64) (string, error) {
+	stringBlock := ""
+	if blockNumber == 0 {
```

Since we process everything block by block, we want this to explicitly be the block height we're processing, i.e. the provided `blockHeight` argument

---

If it's 0, we should return an error

---

It was used 0 in block-lake, so irrelevant here - removing.